src/core/tests/CMakeLists.txt

```diff
@@ -341,6 +341,7 @@ set(SRC
     visitors/op/maximum.cpp
     visitors/op/minimum.cpp
     visitors/op/mish.cpp
+    visitors/op/memory.cpp
```

Comment:
Can we split this file on separate files for each operation?

Reply:
No problem to split the file. Most of the tests for read_value and assign is combined into the one file named by memory. But visitor test is just for each opset, that I think it makes sense to split.